WordPress/Classes/Utility/PushNotificationsManager.swift

```diff
@@ -4,13 +4,6 @@ import WordPressComAnalytics
 
 
 
-/// These notifications are sent when the user Registers / Unregisters for Push Notifications.
-///
-public let NotificationsManagerDidRegisterDeviceToken   = "NotificationsManagerDidRegisterDeviceToken"
-public let NotificationsManagerDidUnregisterDeviceToken = "NotificationsManagerDidUnregisterDeviceToken"
```

Comment:
I tried to spot who was consuming this, (went back to WPiOS Tag 4.0!), but couldn't spot the hook. Orphan code, long lost, i'd say!